src/data-table/measure-column-widths.js

```diff
@@ -152,61 +154,26 @@ export default function MeasureColumnWidths({
     height: 0,
   });
 
-  if (measurementCount.current >= finishedMeasurementCount) {
+  // Remove the measurement nodes after we are done updating our column width
+  if (widthMap.size === columns.length) {
     return null;
   }
 
   return (
     // eslint-disable-next-line jsx-a11y/role-supports-aria-props
     <div className={hiddenStyle} aria-hidden role="none">
-      {sampleRowIndicesByColumn.map((rowIndices, columnIndex) => {
-        const Cell = columns[columnIndex].renderCell;
-        return rowIndices.map(rowIndex => (
-          <ElementMeasurer
-            key={`measure-${columnIndex}-${rowIndex}`}
-            onDimensionsChange={dimensions =>
-              handleDimensionsChange(columnIndex, rowIndex, dimensions)
-            }
-            item={
-              <Cell
-                value={columns[columnIndex].mapDataToValue(rows[rowIndex].data)}
-                isMeasured
-                onSelect={
-                  isSelectable && columnIndex === 0 ? () => {} : undefined
```

Comment:
I believe this should be forwarded to where the cell is being rendered

Reply:
Good call. forwarded it to the header, which has the impact on the size.